### PR TITLE
fix: WT-2059 Update biome version to fix framed logo size in Connect & Bridge

### PR DESCRIPTION
**Pull request by sharonsheah**
# Summary

https://immutable.atlassian.net/browse/WT-2059

Updates biome and design-tokens version which should fix the framed logo size

### CODE REVIEW

File: packages/checkout/widgets-lib/src/widgets/bridge/components/BridgeReviewSummaryStyles.ts [hunk 1]
```diff
@@ -41,6 +41,10 @@ export const walletLogoStyles = (walletName: WalletProviderName) => ({
   padding: walletName === WalletProviderName.PASSPORT ? 'base.spacing.x1' : '',
   backgroundColor: 'base.color.translucent.standard.200',
   borderRadius: 'base.borderRadius.x2',
+  svg: {
+    width: 'base.icon.size.400',
+    height: 'base.icon.size.400',
+  },
```
Comment (glomotion) [inline packages/checkout/widgets-lib/src/widgets/bridge/components/BridgeReviewSummaryStyles.ts:47]: @sharonsheah none of these styles should be required. They all come stock standard with FramedLogo.

File: packages/checkout/widgets-lib/src/widgets/connect/components/WalletItem.tsx [hunk 2]
```diff
@@ -26,9 +26,13 @@ export function WalletItem(props: WalletProps) {
...
         sx={{
           minWidth: 'base.icon.size.500',
-          padding: 'base.spacing.x1',
+          padding: wallet.walletProviderName === WalletProviderName.PASSPORT ? 'base.spacing.x1' : '',
           backgroundColor: 'base.color.translucent.standard.100',
           borderRadius: 'base.borderRadius.x2',
+          svg: {
+            width: 'base.icon.size.500',
+            height: 'base.icon.size.500',
+          },
```
Comment (andrearampin) [inline packages/checkout/widgets-lib/src/widgets/connect/components/WalletItem.tsx:35]:
```
sx={walletItemLogoStyles(walletProviderName)}
```

File: packages/checkout/widgets-lib/src/widgets/bridge/components/BridgeReviewSummaryStyles.ts (hunk 1 above)
Comment (sharonsheah) [reply packages/checkout/widgets-lib/src/widgets/bridge/components/BridgeReviewSummaryStyles.ts]: This actually somehow does affect and makes the icon bigger than if I don't include these styles.

Comment (sharonsheah) [reply packages/checkout/widgets-lib/src/widgets/bridge/components/BridgeReviewSummaryStyles.ts]: With these styles:
image

Without styles:
image

**Review by dreamoftrees — APPROVED** [review]
👍

**Review by dreamoftrees — DISMISSED** [review]
👍

**Review by carmen0208 — DISMISSED** [review]

**Review by CodeSchwert — APPROVED** [review]

**Review by CodeSchwert — DISMISSED** [review]

**Review by dreamoftrees — APPROVED** [review]
👍

**Review by mimi-imtbl — APPROVED** [review]

**Review by CodeSchwert — APPROVED** [review]

**Pull request merged**